### ISSUE TITLE
Getting error

### ISSUE THREAD

**Issue by ofirgalcon** [poundbangbash/ssd_service_program#1]
```Munkireport: ssd_service_program.py Error: Traceback (most recent call last):
  File "/usr/local/munki/preflight.d/ssd_service_program.py", line 110, in <module>
    main()
  File "/usr/local/munki/preflight.d/ssd_service_program.py", line 102, in main
    result['ssd_model'] = ssd_specs[0].get("model")
UnboundLocalError: local variable 'ssd_specs' referenced before assignment
WARNING: Munkireport: ssd_service_program.py return code: 1```

So far on every client I tested on. None of them were the relevant model.

When I open the listing page, the menu goes into mobile mode and keyboard commands stop working.

**Comment by poundbangbash** [comment]
Try pulling latest code. 
Attempted fix at https://github.com/poundbangbash/ssd_service_program/commit/82fa72ddc3ad43b08fedf71d991408c30c58960d

**Comment by poundbangbash** [comment]
Resolved in 82fa72d
